### ISSUE TITLE
Config cloud db

### ISSUE THREAD

**Issue by vikyvilla** [mrvautin/expressCart#72]
Actually I'm new to node can you please help me to configure mongo clouddb in expresscart 
Mongodb: White listed IP and access to all 0.0.0.0 (configured properly)
Same mongo string used in another app is working properly
Error:
![error](https://user-images.githubusercontent.com/22211937/61022455-734eb900-a3c3-11e9-9a6b-4303f8acff14.PNG)

**Comment by mrvautin** [comment]
That looks like your user doesn't have permission to create a collection and/or an index on that collection. The app (specifically `connect-mongodb-session`) creates a `sessions` collection and automatically adds an index on it.

One thing to try is checking for the existence of the `sessions` collection on your Mongo DB and possibly look at either adding indexing role to the Mongodb user or running the app once with an elevated permission user to setup the collections properly.

In summary: This is a Mongodb issue, not expressCart.

**Comment by vikyvilla** [comment]
thanks @mrvautin i fixed the issue
